users/models/identity.py

```diff
@@ -426,13 +428,16 @@ def calculate_stats(self, save=True):
             self.save()
 
     def add_alias(self, actor_uri: str):
-        self.aliases = (self.aliases or []) + [actor_uri]
+        self.aliases = list(set((self.aliases or []) + [actor_uri]))
```

Comment:
Don't think we can use `set` here since the ordering is not guaranteed to be stable.

Reply:
You reference `aliases[0]` a few times in this PR. Probably just need an `if self.aliases and actor_uri in self.aliases: return` or something like it.